public/stylesheets/confirmmail.css

```diff
@@ -0,0 +1,13 @@
+.left{
```

Comment:
Where we got this styles ? Is it ours? We could use foundation or bootstrap here.